src/components/AsideHeader/AsideHeaderContext.ts

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
-import {ASIDE_HEADER_COMPACT_WIDTH} from '../constants';
+import {MenuItem} from '../types';
+import {AsideHeaderInnerProps} from './asideHeaderTypes';
 
-interface AsideHeaderContextType {
-    compact: boolean;
+export interface AsideHeaderContextType extends AsideHeaderInnerProps {
```

Comment:
This context is not intended to drag all the props into it, since it is exported outside. Let's try to explicitly pass props whenever possible, or create a separate context for internal use